collector/lib/ConnTracker.cpp

```diff
@@ -319,4 +323,43 @@ void ConnectionTracker::UpdateIgnoredL4ProtoPortPairs(UnorderedSet<L4ProtoPortPa
     }
   }
 }
+
+// Increment the stat counter matching the connection's characteristics
+inline void ConnectionTracker::ClassifyConnectionStats(Connection conn, ConnectionTracker::Stats& stats) const {
```

Comment:
The name could be misleading, one could get an impression only classification is happening without actually updating the  stats argument. Probably `<Update/Increment>NetworkStats(stats, conn)` would be better, what do you think?